compiler-plugin/src/main/scala/scalapb/package.scala

```diff
@@ -1,11 +1,13 @@
+import com.trueaccord.scalapb.compiler.ProtobufGenerator
 import protocbridge.JvmGenerator
 
 package object scalapb {
   def gen(
     flatPackage: Boolean = false,
     javaConversions: Boolean = false,
     grpc: Boolean = true,
-    singleLineToString: Boolean = false): (JvmGenerator, Seq[String]) =
+    singleLineToString: Boolean = false,
+    collectionType: String = "scala.collection.Seq"): (JvmGenerator, Seq[String]) =
```

Comment:
I prefer that this would be a field-level setting (in scalapb.proto), not a global compiler setting. The use case I see is that a single repeated field in a proto is known to be a set, while the rest of the file stays with the default implementation.